mm2src/coins/utxo/utxo_common.rs

```diff
@@ -1805,6 +1805,21 @@ pub fn min_trading_vol(coin: &UtxoCoinFields) -> MmNumber {
 
 pub fn is_asset_chain(coin: &UtxoCoinFields) -> bool { coin.conf.asset_chain }
 
+pub async fn get_raw_transaction<T>(coin: T, req: RawTransactionRequest) -> RawTransactionResult
```

Comment:
As I can see, `coin` can be `&UtxoCoinFields`. It will reduce the byte code size a bit by avoiding monomorphization.